Submitty/Submitty
# [Feature:InstructorUI] Filter Withdrawn Students

## Why is this change important / necessary?
Closes #11007 
TAs and instructors have no way to see if any students are withdrawn or auditing their course, causing unnecessary worry if they are skipping class / missing assignments.

## What is the new behavior?
There is now an option to filter auditing/withdrawn students on checkpoint/numeric lab and the detail grading pages.

UI Change: This new option is included with other existing toggle buttons such as "View Your Sections" and "Switch to Random Order." Since the location of those buttons restricted the addition of the new "Hide Withdrawn Students" toggle, this PR also refactors the UI by converting the buttons to radios, and moving them to the right where there is more space (see the screenshots below).

## Steps to test
1. login as instructor
2. navigate to the manage students page
3. edit student -> change some several student registration types to withdrawn
4. navigate to a checkpoint/numeric lab, see checkbox
5. navigate to Gradeables -> Grade / Preview Grading -> see filter withdrawn student button
6. test if withdrawn student is filtered, and existing buttons retain functionality

## Before
![TA View before grading 4k ultra](https://github.com/user-attachments/assets/1496cffe-fa63-4598-aada-2f55737523e4)
![Grade released before ta view 4k](https://github.com/user-attachments/assets/c77860e5-c0d4-4bca-b8fc-76047aa3c2b3)

## After
![grade details for TA page gradeable after 4k](https://github.com/user-attachments/assets/835a536b-ddd4-4881-aba9-93c4a00db653)
![Grades Released Lab with the new Option 4k after](https://github.com/user-attachments/assets/8ba42f6d-2d5e-4a04-b88d-6ca28c6a14a4)
